pylib/gyp/generator/make.py

```diff
@@ -724,6 +725,10 @@ def QuoteIfNecessary(string):
         string = '"' + string.replace('"', '\\"') + '"'
     return string
 
+def ReplaceSep(string):
```

Comment:
This is Python.  Functions are snake case and classes are CamelCase.
```suggestion
def replace_sep(string):
```

Reply:
Done. I just followed the naming of `QuoteIfNecessary` 😅

https://github.com/nodejs/gyp-next/blob/82d5d76bdcae91760aab25a582ad726974c99936/pylib/gyp/generator/make.py#L721